src/test/java/hudson/plugins/copyartifact/CopyArtifactTest.java

```diff
@@ -619,6 +631,11 @@ public void testSavedBuildSelector() throws Exception {
     public void testSpecificBuildSelector() throws Exception {
         FreeStyleProject other = createArtifactProject(),
                          p = rule.createFreeStyleProject();
+        ParameterDefinition paramDef = new StringParameterDefinition("FOO","foo");
+        ParametersDefinitionProperty paramsDef = new ParametersDefinitionProperty(paramDef);
+        other.addProperty(paramsDef);
+        CopyArtifactPermissionProperty cpp=new CopyArtifactPermissionProperty("/"+p.getName());
```

Comment:
(about white spaces around `=`)
Though [the Jenkins project doesn't follow code styles strictly|https://wiki.jenkins-ci.org/display/JENKINS/Beginners+Guide+to+Contributing#BeginnersGuidetoContributing-CodeStyle], I highly recommend you to follow a some consistent code style.